apps/www/__registry__/default/example/button-loading.tsx

```diff
@@ -4,9 +4,11 @@ import { Button } from "@/registry/default/ui/button"
 
 export default function ButtonLoading() {
   return (
-    <Button disabled>
-      <Loader2 className="mr-2 h-4 w-4 animate-spin" />
-      Please wait
-    </Button>
+    <div className="cursor-progress">
```

Comment:
Hi hope ur well @M4xymS, I tried that approach because I know it's better, but the logic didn't work due to the button's design. So, this is the best solution.